go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/grpc-ecosystem/go-grpc-middleware v1.1.0 // indirect
 	github.com/jetstack/cert-manager v0.6.2 // indirect
 	github.com/metal-stack/firewall-controller v0.1.0
-	github.com/metal-stack/metal-go v0.3.2
+	github.com/metal-stack/metal-go v0.7.8
```

Comment:
Please update to the most recent version. Also here, we still refer to a ancient mcm:



```github.com/gardener/machine-controller-manager => github.com/metal-pod/machine-controller-manager v0.0.0-20190801141331-4e2b75ebc6c0```

This must be fixed as well